system/react/src/components/Button.ts

```diff
@@ -15,9 +15,11 @@ export const classlessSelector = 'button, a[role="button"]';
 export const classySelector = '.btn';
 
 export const baseStyles = css`
+  --padding-y: 12px;
+  --padding-x: 20px;
   position: relative;
   display: grid;
-  padding: 12px 20px;
+  padding: calc(12px - 2px) calc(20px - 2px);
```

Comment:
Calc is right for this, you can use CSS properties in calc.

```suggestion
  padding: calc(var(--padding-y) - 2px) calc(var(--padding-x) - 2px);
```